js/util.js

```diff
@@ -5,10 +5,48 @@ app.util = (function() {
 	'use strict';
 
 	return {
+
 		extend: function() {
+
 			var args = Array.prototype.slice.call(arguments);
 			return _.extend.apply(_, [{}].concat(args));
+		},
+
+		generateRandomString: function(length, charset) {
+
+			var randString = '';
+
+			charset = charset || 'abcdefghijklmnopqrstuvqxyzABCDEFGHIJKLMNOPQRSTUVQXYZ1234567890';
+
+			var randomValues = app.util.getRandomValues(length);
+
+			for (var index = 0; index < length; index++) {
+				randString += charset[randomValues[index] % charset.length];
+			}
+
+			return randString;
+		},
+
+		getRandomValues: function(length) {
+
+			var randomValues;
+
+			// This should work for the latest browsers:
+			if (window.crypto && window.crypto.getRandomValues) {
```

Comment:
Is the crypto library in the browser random enough?

Reply:
I think using `window.crypto` is actually better than using the sjcl library.